data/src/main/java/com/example/data/service/utils/Constants.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.data.service.utils
+
+object Constants {
```

Comment:
you should unify the constants in the domain layer, as you are repeating the constants